frontend/app/src/locales/en.json

```diff
@@ -727,7 +727,7 @@
   "no_filter_available": {
     "hint": {
       "description": "* type the filter and then press enter e.g.",
-      "example": "quote: ETH"
```

Comment:
Changing it since previous suggestion was relevant only for `Trades` page because there is no filter `quote:` on `Deposits & Withdrawals` nor `Ethereum Transactions` nor `Ledger Actions`.

Reply:
This change/suggestion is what actually made the e2e tests run and they failed. Not sure if it's due to that, but I guess it can't be, and it's just flakiness.

@kelsos  ?

Reply:
Which test failed?

Reply:
ARGS="-s tests/e2e/specs/accounts-balances.spec.ts" npm run test:integration-ci -w rotki

Reply:
resolving this as from what I understand this does not affect the app or tests in any way